### DISCUSSION TITLE
Saved Replies

### DISCUSSION THREAD

**Discussion by fishcharlie** [community/community#6967] [Mobile]
It would be awesome if using Saved Replies was possible on the GitHub mobile app.

**Comment by brianlovin** [comment]
Agreed, I've added a +1 to our tracking issue for this — thanks @fishcharlie!

**Comment by jsoref** [comment]
This feature appears available on Android:
![Screenshot_20240721-160632.png](https://github.com/user-attachments/assets/11a09fb5-fa85-406e-b4a4-3737bbbfff12)
